scalikejdbc-core/src/test/scala/scalikejdbc/StatementExecutorSpec.scala

```diff
@@ -15,4 +17,16 @@ class StatementExecutorSpec extends FlatSpec with Matchers {
     instance should not be null
   }
 
+  it should "print sql string" in {
+    val underlying: PreparedStatement = mock[PreparedStatement]
+    val template: String = "select id, name from members where id = ? and name = ?"
+    val params: Seq[Any] = Seq(1, "name1")
+    val instance = new StatementExecutor(underlying, template, DBConnectionAttributes(), params)
+    val runtimeMirror = ru.runtimeMirror(instance.getClass.getClassLoader)
+    val instanceMirror = runtimeMirror.reflect(instance)
+    val method = ru.typeOf[StatementExecutor].member(ru.newTermName("sqlString")).asMethod
```

Comment:
```
scalikejdbc-core/src/test/scala/scalikejdbc/StatementExecutorSpec.scala:27:57: method newTermName in trait Names is deprecated (since 2.11.0): use TermName instead
[warn]     val method = ru.typeOf[StatementExecutor].member(ru.newTermName("sqlString")).asMethod
[warn]                                                         ^
[warn] two warnings found
```

Reply:
Using `TermName` instead of `newTermName` in Scala 2.10.6 causes a compile error.
https://travis-ci.org/scalikejdbc/scalikejdbc/builds/332268974
https://travis-ci.org/scalikejdbc/scalikejdbc/jobs/332268975#L633

Now I confirmed #818 merged, I try to fix it.